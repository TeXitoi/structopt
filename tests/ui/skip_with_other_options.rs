use structopt::StructOpt;

#[derive(StructOpt, Debug)]
#[structopt(name = "test")]
pub struct Opts {
    #[structopt(long)]
    a: u32,
    #[structopt(skip, long)]
    b: u32,
}

fn main() {
    let opts = Opts::from_args();
    println!("{:?}", opts);
}
