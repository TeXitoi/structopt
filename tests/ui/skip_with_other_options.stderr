error: proc-macro derive panicked
 --> $DIR/skip_with_other_options.rs:3:10
  |
3 | #[derive(StructOpt, Debug)]
  |          ^^^^^^^^^
  |
  = help: message: methods are not allowed for skipped fields
